client/jetpack-onboarding/steps/summary.jsx

```diff
@@ -95,9 +99,15 @@ class JetpackOnboardingSummaryStep extends React.PureComponent {
 	}
 }
 
-export default connect( () => {
-	const tasks = compact( [] );
-	return {
-		tasks,
-	};
-} )( localize( JetpackOnboardingSummaryStep ) );
+export default connect( ( state, { siteId } ) => ( {
+	hasWooCommerceInstallFinished: get(
+		getRequest(
+			state,
+			saveJetpackOnboardingSettings( siteId, {
+				installWooCommerce: true,
+			} )
+		),
+		'hasLoaded'
+	),
```

Comment:
note that this `get()` is redundant: `getRequest()` I believe will always return an object and even further, `hasLoaded` will always be valid.